samples/subsys/usb/mass/Kconfig

```diff
@@ -45,6 +51,17 @@ config DISK_RAM_VOLUME_SIZE
 config MASS_STORAGE_DISK_NAME
 	default "NAND" if DISK_ACCESS_FLASH
 	default "RAM" if DISK_ACCESS_RAM
+	default "SD" if DISK_ACCESS_SDHC
+
+if DISK_ACCESS_SDHC
+
+config SPI
+	default y
+
+config DISK_ACCESS_SPI_SDHC
+	default y
+
+endif #DISK_ACCESS_SDHC
```

Comment:
Can you move this to a board overlay? I tested this PR on the `mimxrt1050_evk` board, which uses the USDHC driver instead of the SPI driver. It works, but there are build warnings:
```
/home/maureen/zephyrproject/zephyr/subsys/disk/disk_access_spi_sdhc.c:28:2: warning: #warning NO SDHC slot specified on board [-Wcpp]
   28 | #warning NO SDHC slot specified on board
      |  ^~~~~~~
```

Reply:
Using board config overlays with this example is not good. With all the different configuration choices, Kconfig is the better option. I have changed sample Kconfig so that DISK_ACCESS_SPI_SDHC is taken into account.